tequilapi/client/dto.go

```diff
@@ -25,14 +27,18 @@ type ProposalDTO struct {
 	ServiceDefinition ServiceDefinitionDTO `json:"serviceDefinition"`
 }
 
+func (p ProposalDTO) String() string {
+	return fmt.Sprintf("Id: %d , Provider: %s, Country: %s", p.ID, p.ProviderID, p.ServiceDefinition.LocationOriginate.Country)
```

Comment:
```go
p.ServiceDefinition.LocationOriginate.Country
```
This is very long call chain and it seems to be repeated everywhere we need country. Perhaps service proposal or service definition could have `country` shortcut method?